### PR TITLE
Add seeker statuses

### PR DESCRIPTION
**Pull request by mkcp**
Here we:
- Declares the `seeker.Status` type and 3 `Status`es e.g.: `[]Status{Success, Fail, Unknown}`
- Add `Status` as to the return values of `Runner.Run()`
- Add `Status` to the `Seeker` struct and write it in `Seeker.Run()`
- Migrate existing seeker implementations to return a `Status`.

Following this work, we want to read each status from seekers in each Product, and report the sum of a product's statuses  via logs, Manifest, and Results.

### CODE REVIEW

File: seeker/commander.go [hunk 1]
```diff
@@ -22,29 +22,38 @@ func NewCommander(command string, format string) *Seeker {
...
+func (c Commander) Run() (interface{}, Status, error) {
+	var result interface{}
+	var err error
+
 	bits := strings.Split(c.Command, " ")
 	cmd := bits[0]
 	args := bits[1:]
 
-	bts, err := exec.Command(cmd, args...).CombinedOutput()
+	// TODO(mkcp): Add cross-platform commandExists() func to ensure there's a bin we can call
```
Comment (mkcp) [inline seeker/commander.go:34]: Intending to create a Jira and delete this TODO before merge.

File: seeker/commander.go [hunk 2]
```diff
@@ -22,29 +22,38 @@ func NewCommander(command string, format string) *Seeker {
...
 
+	// Execute command
+	bts, err := exec.Command(cmd, args...).CombinedOutput()
 	if err != nil {
-		err = fmt.Errorf("exec.Command error: %s", err)
+		return string(bts), Unknown, fmt.Errorf("exec.Command error: %s", err)
 	}
 
+	// Parse result
+	// TODO(mkcp): This can be detected rather than branching on user input
```
Comment (mkcp) [inline seeker/commander.go:43]: Intending to create a Jira and delete this TODO before merge.

File: seeker/seeker.go [hunk 3]
```diff
@@ -5,24 +5,36 @@ import (
 	"path/filepath"
 )
 
+// status describes the result of a seeker run
+type Status string
+
+const (
+	Success Status = "success"
+	Fail    Status = "fail"
+	Unknown Status = "unknown"
```
[the remark this answers is not in the document]
Comment (mkcp) [reply seeker/seeker.go:14]: Yeah that would be good to add - I'll write something up.

[the remark this answers is not in the document]
Comment (nwchandler) [reply seeker/seeker.go:19]: Thanks!

**Review by nwchandler — APPROVED** [review]
Looks good to me

**Pull request merged**